draft-ietf-quic-transport.md

```diff
@@ -2291,10 +2291,10 @@ Too Big messages.
 provisional until QUIC's loss detection algorithm determines that the packet is
```

Comment:
I would like to see some discussion of unilateral and GOAWAY frame. The current text describing that frame assumes bilateral streams.